mlir/lib/Target/CppOutput/gridwise_convolution_implicit_gemm_v4r4.cpp

```diff
@@ -244,7 +330,7 @@ static constexpr StringLiteral kCppInterludeFormat = R"(
     using GemmBBlockCopyThreadClusterLengths_GemmK_GemmN =
```

Comment:
Has MIOpen v4r4 and v1r1 source code been changed at all over the past couple of months? I will assume yes, and skip reviewing the cpp/header generation logic, since source level differences between codegen and MIOpen are easy to find.